core/src/test/java/org/elasticsearch/search/sort/SortParserTests.java

```diff
@@ -47,7 +47,7 @@ public void testGeoDistanceSortParserManyPointsNoException() throws Exception {
         sortBuilder.endArray();
         sortBuilder.field("order", "desc");
         sortBuilder.field("unit", "km");
-        sortBuilder.field("sort_mode", "max");
+        sortBuilder.field("mode", "max");
```

Comment:
oh these changes seem to be a side effect of setting STRICT somewhere? But I thought we had STRICT already in the base class parser. Which change is causing this then?